edenia/website-boilerplate
# build(yarn): update version

### Update version

### What does this PR do?

- Resolve #issueNumber

### Steps to test

1. Run project
2. Check changes

Before:
![imagen](https://github.com/edenia/crservers.com/assets/49013556/a2deced2-fb99-4c1d-84b7-751683e6170c)

Now:
![imagen](https://github.com/edenia/crservers.com/assets/49013556/2d39807a-87c3-41c1-bde5-34717c36d757)


Base on:
https://github.com/nuxt/image/issues/1227
https://yarnpkg.com/migration/guide

#### CheckList
